### PR TITLE
security: Add strongSwan container to PoC

### PR DESCRIPTION
**Pull request by mestery**
This adds a strongSwan container [1] [2] to the OPI PoC. Step one of
testing the new OPI Security APIs.

[1] https://hub.docker.com/r/strongx509/strongswan
[2] https://github.com/strongX509/docker

Related to #220

Signed-off-by: Kyle Mestery <mestery@mestery.com>

### CODE REVIEW

**Comment by glimchb** [comment]
let's do a separate security folder? like storage ?

**Comment by mestery** [comment]
> let's do a separate security folder? like storage ?

We will do that for our gRPC server for security, but this is just a straight strongSwan container, similar to SPDK. So the config should live here.

File: integration/docker-compose.xpu.yml [hunk 1]
```diff
@@ -98,6 +98,20 @@ services:
       start_period: 20s
       timeout: 10s
 
+  strongswan:
+    image: strongx509/strongswan:5.9.6
+    cap_add:
+      - NET_ADMIN
+      - SYS_ADMIN
+      - SYS_MODULE
+    stdin_open: true
```
Comment (glimchb) [inline integration/docker-compose.xpu.yml:107]: maybe we can fiond a way without those... 
```
    stdin_open: true
    tty: true
```

**Review by glimchb — APPROVED** [review]
looks like a start

**Comment by glimchb** [comment]
some addition to `integration/scripts/integration.sh` would be nice, can be in the next PR

**Comment by mestery** [comment]
> some addition to `integration/scripts/integration.sh` would be nice, can be in the next PR

I will update the issue to track the changes to the testing. Thanks for the reminder!

File: integration/strongswan/server/ecdsa/serverKey.pem [hunk 2]
```diff
@@ -0,0 +1,6 @@
+-----BEGIN EC PRIVATE KEY-----
```
Comment (seroyer) [inline integration/strongswan/server/ecdsa/serverKey.pem:1]: Seeing private keys, passwords, API keys, etc put into a public github repo always makes me twitch.  I _know_ that this is a horrible practice, and I keep having to convince myself that it's ok for integration test / poc...

**Comment by mestery** [comment]
> some addition to `integration/scripts/integration.sh` would be nice, can be in the next PR

Added some quick sanity tests.

File: integration/scripts/integration.sh [hunk 3]
```diff
@@ -79,6 +79,8 @@ run_integration_tests() {
     bash -c "${DC} exec -T spdk-target /usr/local/bin/perf     -r 'traddr:10.129.129.4 trtype:TCP adrfam:IPv4 trsvcid:4420' -c 0x1 -q 1 -o 4096 -w randread -t 10"
     bash -c "${DC} exec -T xpu-spdk /usr/local/bin/perf         -r 'traddr:10.129.129.4 trtype:TCP adrfam:IPv4 trsvcid:4420' -c 0x1 -q 1 -o 4096 -w randread -t 10"
     bash -c "${DC} up example-storage-client example-network-client"
+    bash -c "${DC} exec -T strongswan swanctl --stats"
+    bash -c "${DC} exec -T strongswan swanctl --list-sas"
 }
```
Comment (glimchb) [inline integration/scripts/integration.sh:84]: great!

**Pull request merged**